tests/widgets/test_colour_text.py

```diff
@@ -0,0 +1,119 @@
+# Copyright (C) 2017  Lucas Hoffmann
+# This file is released under the GNU GPL, version 3 or a later revision.
+# For further details see the COPYING file
+
+import unittest
+from urwid import AttrSpec
+from unittest import mock
+
+from alot.settings.const import settings
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unused settings imported from alot.settings.const](https://app.codacy.com/manual/patricktotzke/alot/pullRequest?prid=4429219)

Reply:
Removed it in 32cebe8